### ISSUE TITLE
Line chart: Not able to focus all the points in chart, while using browse mode in NVDA

### ISSUE THREAD

**Issue by jameelakowsar** [microsoft/fluentui#16124]
### Environment Information

- **Package version(s)**: Above 4 versions
- **Browser and OS versions**:  Edge

#### Actual behavior:

In browse mode, Not able to access all points in charts

#### Expected behavior:

All elements in charts should be accessible in browse mode.

Repro link: https://admincontrolsdemoapps.blob.core.windows.net/release/charting-demo/3.0.1/index.html#/examples/LineCharts

steps: 
1. Open above link
2. Turn on NVDA 
3. While navigating to lines, turn on browse mode. See that, all points at lines not able to access and NVDA is announcing 'clickable clickable clickable...continuously'. 

Note:
1) Issue repro only using NVDA in all browsers
2) In browse mode, Legends are also not accessible.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by JustSlone** [comment]
@jameelakowsar, can you provide a CodePen and some repro steps? It will help fix the issue. 

@Raghurk, FYI on this issue in charting
